### PR TITLE
Belief management and topic reading in database

### PR DESCRIPTION
**Pull request by sdevin**

### CODE REVIEW

File: database_manager/src/run_server.cpp [hunk 1]
```diff
@@ -2575,17 +2569,39 @@ int main(int argc, char **argv) {
 
 
     initServer();
-
-    //ToasterFactReader factRdPdg(node, "pdg/factList");
```
Comment (Greg8978) [inline database_manager/src/run_server.cpp:2579]: Why did you remove factList from pdg?

**Comment by Greg8978** [comment]
Why did you remove facts from pdg?
I think pdg export some facts like isSeen or hasInHand...

Otherwise it's good. I didn't look in details for the perspective taking part but anyways it was not there so it can only be an improvement ^^

**Comment by sdevin** [comment]
I just forgot to re-add it^^
Its corrected.

**Comment by Greg8978** [comment]
Thank you :+1:
